appengine/flexible/sparkpost/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==0.12
+gunicorn==19.7.1
+sparkpost==1.3.5
```

Comment:
Those test requirements are not included in the project because I believe they form part of a [repo-wide system test setup](https://github.com/ewandennis/python-docs-samples/blob/master/TESTING.md).